src/hatch_openzim/utils.py

```diff
@@ -0,0 +1,64 @@
+import re
+import subprocess
+from functools import lru_cache
+from typing import List
+
+from packaging.specifiers import SpecifierSet
+from packaging.version import Version
+
+REMOTE_REGEXP = re.compile(
+    r"""^(?:git@|https:\/\/)github.com[:\/](.*?)\/(.*?)(?:.git)?$"""
+)
+
+
+@lru_cache(maxsize=None)
```

Comment:
Is this called multiple times?

Reply:
No, you are right, this is premature optimization